opennms-doc/guide-development/src/asciidoc/text/dev-guide.adoc

```diff
@@ -0,0 +1,96 @@
+
+This guide describes the requirements and the steps necessary in order to get started with the development of the OpenNMS project.
+
+=== Operating System / Environment
+
+To build/compile OpenNMS it is necessary to run a *nix system.
+You do not need to run it physically, a virtual machine is sufficient, but the choice is yours.
+We recommend one of the following:
+
+ * link:http://www.linuxmint.com/[Linux Minut] with Cinnamon Desktop environment
+ * link:http://ubuntu.com[Ubuntu Desktop]
+ * Mac OS X
+
+[NOTE]
+This documentation assumes that you chose a debian based desktop environment.
+
+=== Installation
+
+The next chapter describes the full setup of your environment in order to meet the pre-requirements.
+Simply follow these instructions, they may vary depending on your Operating System.
+
+[source, shell]
+----
+# add OpenNMS as repository to install icmp and such
```

Comment:
Here we need to add the installation of maven and the Oracle JDK 8, because that is completly missing at this point.